Readme-developer.md

```diff
@@ -1,4 +1,4 @@
-Building on OS X 10.9 & 11.4
+Building from source
```

Comment:
I believe there is Readme.md that has information for not-osx operation systems., maybe we could move your additions to a separate readme? Or probably put it in that readme?

Reply:
I will have you to decide that - what is the best?

Reply:
@mhrvol

Reply:
I will add this to the readme in dx-toolkit/src/python/Readme.md - Building section